src/renderer/components/ft-button/ft-button.js

```diff
@@ -18,6 +18,11 @@ export default defineComponent({
     id: {
       type: String,
       default: ''
-    }
+    },
+    tabindex: {
+      type: Number,
+      // Relies on template to not output tabindex on values < -1
+      default: -2,
+    },
```

Comment:
**question:** Why was this added?

Reply:
Probably part of the workaround I introduced before
Let me see if they can be removed

Reply:
Removed
Probably wanted to set as `-1` at some point as workaround to some issue which can no longer be reproduced
Forgot what issue